_datafiles/localize/en.yaml

```diff
@@ -0,0 +1,15 @@
+# English
+
+User.Id: 'UserId'
+User.Name: 'Name'
```

Comment:
Curious on the rationale for these...

Why `User.Id` instead of `UserId` or `User.UserId`?
Why `User.Name` instead of `Username`?

Reply:
`UserId` is OK, for `User.Name` it is because the `Name` used in the original field is ambiguous, it is not possible to distinguish whether it is `User Name` or `Room Name`, etc.

The same words or phrases may have different meanings in different scenarios, e.g. for `Login.username` and `Login.password` in the login template, it may be necessary to go beyond simply translating `username` and `password`, and of course modifying the original text is also an option.

So either we don't use ambiguous characters, or we need to add additional prefixes to avoid ambiguity, what do you think of this?

Reply:
I think the prefixes make sense, thanks.